src/service/resource.js

```diff
@@ -509,6 +514,14 @@ export class Resource {
   }
 
   /**
+   * Whether the element is in the first viewport ("above the fold").
+   * @return {boolean}
+   */
+  isInFirstViewport() {
+    return this.isInFirstViewport_;
```

Comment:
I assume `#isInFirstViewport` should always be called after element's measurement. How are we making sure that this is the case?